pkg/oc/cli/admin/migrate/scc/scc.go

```diff
@@ -278,3 +303,46 @@ func convertSccToClusterRoleBinding(scc *securityapi.SecurityContextConstraints)
 
 	return &binding, nil
 }
+
+func convertClusterRolesWithSCCRules(scc *securityapi.SecurityContextConstraints, roles *rbacv1.ClusterRoleList) (rolesToUpdate []rbacv1.ClusterRole) {
+	for roleIt := 0; roleIt < len(roles.Items); roleIt++ {
```

Comment:
Why it's `roleIt` and not just `i`?

Reply:
I moved from a prototype which was terrible and had several nested for cycles. I'll rename the variable.

Reply:
ok, np. Fortunately, we have a code review! :)